components/Author/Feed/types/forYouFeedTypes.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Paper,
+  Post,
+  parsePaper,
+  parsePost,
+} from "~/components/Document/lib/types";
+import Bounty from "~/config/types/bounty";
+import { parseContentType } from "~/config/types/contentType";
+import { Hub, parseHub } from "~/config/types/hub";
+import {
+  ID,
+  RHUser,
+  UnifiedDocument,
+  parseUser,
+} from "~/config/types/root_types";
+
+export interface ForYouFeedItem {
+  item: Paper | Post | Bounty | ForYouFeedCommentObject;
+  contentType: string;
+
+  hubs: Hub[];
+  unifiedDocument: UnifiedDocument;
+
+  userVote: number | null;
+  score: number;
+
+  createdBy: RHUser;
+  createdDate: string;
+}
+export interface ForYouFeedPaperItem extends ForYouFeedItem {
+  item: Paper;
+}
+export interface ForYouFeedPostItem extends ForYouFeedItem {
+  item: Post;
+}
+export interface ForYouFeedBountyItem extends ForYouFeedItem {
+  item: Bounty;
+}
+export interface ForYouFeedCommentItem extends ForYouFeedItem {
+  item: ForYouFeedCommentObject;
```

Comment:
My guess is that you created this type because some data is missing? If so, I can include the additional missing so that we don't have to create this new type

Reply:
Yeah that is why, it didn't fit within the Comment parser or the CommentContribution parser